src/index.js

```diff
@@ -432,7 +421,9 @@ export class Xapi extends EventEmitter {
 
     // http-request-plus correctly handle redirects if body is not a stream
     if (typeof body.pipe !== 'function') {
-      return doRequest($cancelToken)
+      return doRequest($cancelToken).then(response => {
```

Comment:
You can't pass a cancel token to `doRequest` anymore, right?
And you'd also need to remove `null` in the parameters line 430 (thanks Github for not letting me write a comment there).